src/software/new_geom/convex_polygon.cpp

```diff
@@ -27,6 +27,16 @@ bool ConvexPolygon::isConvex()
         return false;
     }
 
+    // Degenerate polygons are convex
```

Comment:
Could we refactor this into `isDegenerate` function?

Reply:
done